src/test/run-pass/existential_type_const.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(existential_type)]
```

Comment:
Please move this to `ui/existential_types/` and use a `// check-pass` annotation since there are no dynamic semantics under test. The same should apply to the other new test as well.